packages/lesswrong/lib/collections/posts/helpers.js

```diff
@@ -195,3 +195,13 @@ Posts.getKarma = (post) => {
   const baseScore = getSetting('forumType') === 'AlignmentForum' ? post.afBaseScore : post.baseScore
   return baseScore || 0
 }
+
+// User can add/edit the hideCommentKarma setting if:
+//  1) The user has the requiset setting enabled
+//  And
+//  2) The post does not exist yet
+//  Or if the post does exist
+//  3) The post doesn't have any comments yet
+Posts.canEditHideCommentKarma = (user, post) => {
+  return user['showHideKarmaOption'] && (!post || !Posts.getCommentCount(post))
```

Comment:
Why this really weird "array with string" way of referencing that field on the user object?

Reply:
Cause I programmed in python too recently.